react-app/src/components/molecules/MovieCard/MovieCardComponent.tsx

```diff
@@ -17,9 +18,12 @@ import {
 } from './MovieCard';
 import { actionControlVisibility, actionSelectMovie } from '../../../actions/Actions';
 
+// PATTERN: render prop
```

Comment:
Aren't you suppose to pass in some dynamically rendered prop-function to implement render props?

Reply:
Props item passed inside.